iosTests/package.json

```diff
@@ -11,25 +11,25 @@
   "dependencies": {
     "create-react-class": "^15.7.0",
     "react": "18.2.0",
-    "react-native": "0.72.7",
-    "react-native-force": "git+https://github.com/forcedotcom/SalesforceMobileSDK-ReactNative.git#dev"
+    "react-native": "0.73.6",
+    "react-native-force": "git+https://github.com/wmathurin/SalesforceMobileSDK-ReactNative.git#rn73"
```

Comment:
Pointing to wmathurin rn73 to allow tests to pass. Will revert that line before merging.